.changeset/mighty-humans-tan.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-scaffolder-backend-module-gerrit': minor
```

Comment:
```suggestion
'@backstage/plugin-scaffolder-backend-module-gerrit': patch
```

Bringing it down to `patch` since there's no breaking changes here for a `0.x` package